02-props-state-component-architecture/props/pokedex/src/pokedex.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import Pokecard from './pokecard';
+
+
+class Pokedex extends Component{
+	render(){
```

Comment:
just a note, in the future this can be a stateless functional component since all it does is `render()`. I don't think we taught you before you did this exercise tho 😉